src/tests/integration/konvert/test_simple_proofs.py

```diff
@@ -283,6 +283,49 @@
         'kseq{}(VarELEM1 : SortKItem{}, kseq{}(VarELEM2 : SortKItem{}, dotk{}()))',
         KSequence([KVariable('ELEM1', sort=KSort('KItem')), KVariable('ELEM2', sort=KSort('KItem'))]),
     ),
+    (
+        'if-then-else-no-sort-param-k',
+        KSort('K'),
+        r"""
+        Lbl'Hash'if'UndsHash'then'UndsHash'else'UndsHash'fi'Unds'K-EQUAL-SYNTAX'Unds'Sort'Unds'Bool'Unds'Sort'Unds'Sort{SortK{}}(
+            VarC : SortBool{}, VarB1 : SortK{}, VarB2 : SortK {}
+        )
+        """,
+        KApply(
+            KLabel('#if_#then_#else_#fi_K-EQUAL-SYNTAX_Sort_Bool_Sort_Sort', []),
+            [
+                KVariable('C', sort=KSort('Bool')),
+                KVariable('B1', sort=KSort('K')),
+                KVariable('B2', sort=KSort('K')),
+            ],
+        ),
```

Comment:
You can make these more compact:

```suggestion
        KApply(
            KLabel('#if_#then_#else_#fi_K-EQUAL-SYNTAX_Sort_Bool_Sort_Sort'),
            KVariable('C', sort='Bool'),
            KVariable('B1', sort='K'),
            KVariable('B2', sort='K'),
        ),
```